jsonc/parse_test.ts

```diff
@@ -8,6 +8,10 @@ import {
   assertThrows,
 } from "@std/assert";
 
+import "./testdata/JSONTestSuite/test.ts";
+import "./testdata/node-jsonc-parser/test.ts";
+import "./testdata/test262/test.ts";
```

Comment:
I think these test cases used to run without these imports, but currently they are not discovered by `deno test` command.